types/ibc.go

```diff
@@ -133,16 +133,19 @@ func (m *IBCCloseConfirm) ToMsg() IBCChannelCloseMsg {
 }
 
 type IBCPacketReceiveMsg struct {
-	Packet IBCPacket `json:"packet"`
+	Packet  IBCPacket `json:"packet"`
+	Relayer string    `json:"relayer"`
```

Comment:
Sorry if I confused you, but isn't omitempt required or at least useful to support v1 and v3 at the same time? In v1 the field should not be there. In v3 it is always set.

I guess both ways work. Why did you change it?